src/content/en/fundamentals/instant-and-offline/get-started/wrapup-7.md

```diff
@@ -0,0 +1,541 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2018-01-25 #}
+{# wf_published_on: 2018-01-25 #}
+
+# Wrap-up and Demo {: .page-title }
+
+{% include "web/_shared/contributors/dgash.html" %}
+
+
+We've now explored various ways to improve page load time. These techniques are "low-hanging 
+fruit" -- simple efforts that produce big performance wins. But talk is cheap; we have not 
+yet actually seen the techniques in action, so finish up by doing that.
+
+To demonstrate the techniques' effectiveness, we will run various speed tests on different 
+versions of a web page, beginning with no optimization and progressing through some of 
+the speed-up techniques we've discussed, specifically text resource optimization, 
+graphics resource optimization, and HTTP request reduction.
+
+The original target is a simple, single HTML page with text, images, CSS, and JavaScript, 
+hosted on Firebase here: 
+[https://page-load-speed-demo.firebaseapp.com/pageload0.html](https://page-load-speed-demo.firebaseapp.com/pageload0.html).
```

Comment:
@Meggin As we're referencing this demo app in official guidance, it should probably be open-sourced somewhere the rest of the team can update and maintain over time. Can you work with Dave to perhaps land it in GoogleChrome/samples or its own repo?